code/__DEFINES/research/research_categories.dm

```diff
@@ -179,7 +179,7 @@
 #define RND_SUBCATEGORY_IMPLANTS "/Implants"
 
 // Limb Categories
-#define RND_CATEGORY_LIMBS_DIGITIGRADE "Digitigrade"
+#define RND_CATEGORY_LIMBS_DIGITIGRADE "digitigrade"
```

Comment:
why

Reply:
having it start with a capital fucks up the typepath and icon of the created limb since it uses the category name to text2path the limb it makes and uses it when naming the limb

Reply:
thats stupid as hell